src/App.js

```diff
@@ -14,15 +14,16 @@ import Blogs from './containers/Blogs/Blogs';
 import NotFound from './containers/NotFound/NotFound';
 import Navbar from './components/Navbar/Navbar';
 import Footer from './components/Footer/Footer';
-
+import SingleWorkout from './containers/SingleWorkout/SingleWorkout';
 const DefaultRoutes = () => {
   return (
     <>
       <Navbar />
       <Switch>
         <Route path="/home" component={Home} />
         <Route path="/meals" component={Meals} />
-        <Route path="/workouts" component={Workouts} />
+        <Route exact path="/workouts" component={Workouts} />
+        <Route path="/workouts/singleworkout" component={SingleWorkout} />
```

Comment:
its better to send the is of the workout so you can get its information in that page
```suggestion
        <Route path="/workouts/:id" component={SingleWorkout} />
```